packages/database/src/migrations/20210708023241-LESMISListVisualSummaryReport-modifies-data.js

```diff
@@ -0,0 +1,357 @@
+      "'parent'": "'IO1_1'",
+    },
+    {
+      transform: 'insert',
+      position: 'before',
+      where: 'eq($index, 1)',
+      "'code'": "'Strategy4'",
+      "'label'":
+        "'Strategy 4: Expansion of WASH facilities and program at ECE level, particularly for girls and children with disabilities'",
+      "'parent'": "'IO1_1'",
+    },
+    {
+      transform: 'insert',
+      position: 'before',
+      where: 'eq($index, 1)',
+      "'code'": "'Strategy3'",
+      "'label'":
+        "'Strategy 3: Development of a sustainable and cost-effective (human and financial resourcing) targeted ECE school meals strategy and action plan'",
+      "'parent'": "'IO1_1'",
+    },
+    {
+      transform: 'insert',
+      position: 'before',
+      where: 'eq($index, 1)',
+      "'code'": "'Strategy2'",
+      "'label'":
+        "'Strategy 2: Scaling up of proven innovations/pilots for 3-5-year olds, with priority on programmes for 5-year olds, such as multi-age teaching (more research may be needed)'",
+      "'parent'": "'IO1_1'",
+    },
+    {
+      transform: 'insert',
+      position: 'before',
+      where: 'eq($index, 1)',
+      "'code'": "'Strategy1'",
+      "'label'":
+        "'Strategy 1: Prioritised and targeted expansion of the one-year pre-primary program subject to quality improvements to provide more access in smaller villages from more disadvantaged areas.'",
+      "'parent'": "'IO1_1'",
+    },
+    {
+      transform: 'insert',
+      position: 'before',
+      where: 'eq($index, 1)',
+      "'code'": "'IO1_1'",
+      "'label'": "'IO 1.1: Increased access to quality ECE (Part I HLO 1, IOs 1.2 and 1.3)'",
+      "'parent'": "'HLO1'",
+      "'statistic'":
+        'eq(max($where(f($otherRow) = equalText($otherRow.parent, "IO1_1")).statistic), min($where(f($otherRow) = equalText($otherRow.parent, "IO1_1")).statistic)) ? last($where(f($otherRow) = equalText($otherRow.parent, "IO1_1")).statistic) : 0',
+    },
+    {
+      transform: 'insert',
+      position: 'before',
+      where: 'eq($index, 1)',
+      "'code'": "'HLO1'",
+      "'label'":
+        "'HLO 1: Increased number of ECE graduates with improved primary school readiness, particularly literacy and numeracy skills with special focus on disadvantaged and gender equity'",
+      "'statistic'":
+        'eq(max($where(f($otherRow) = equalText($otherRow.parent, "HLO1")).statistic), min($where(f($otherRow) = equalText($otherRow.parent, "HLO1")).statistic)) ? last($where(f($otherRow) = equalText($otherRow.parent, "HLO1")).statistic) : 0',
+    },
+  ],
+};
+
+const FRONT_END_CONFIG = {
+  name: 'ESSDP List Summary',
+  type: 'list',
+  valueType: 'color',
+  listConfig: {
+    Green: {
+      color: '#4caf50',
+      value: 1,
+    },
+    Yellow: {
+      color: '#ffeb3b',
+      value: 0,
+    },
+    Red: {
+      color: '#f44336',
+      value: -1,
+    },
```

Comment:
Hmm, I think this might work better with the value as the key. Generally the front end will be interested in asking "for this value, what colour should I use?" (and maybe "what hover text", which would involve the `Green` etc. moving in under a `name` or `hover` or `label` key)